src/opendlv-statsd.cpp

```diff
@@ -47,27 +59,90 @@ int32_t main(int32_t argc, char **argv) {
         //       would have to be created and registered.
 
         // Handler to receive a GPS message (realized as C++ lambda).
-        auto onGPS = [&dataMutex](cluon::data::Envelope &&env){
+        
+        std::vector<double> lat_queue,lon_queue; //vector is used as a queue
+        
+        auto onGPS = [&dataMutex,&lat_queue,&lon_queue,VERBOSE](cluon::data::Envelope &&env){
```

Comment:
This lambda function is tied to the GPS example; how would the message-agnostic approach look like?